src/main/java/com/RegisterQuizApplication/Registration/Choice.java

```diff
@@ -0,0 +1,40 @@
+package com.RegisterQuizApplication.Registration;
```

Comment:
package name should always be lowercase. Also package names should be meaningful.

- com.quiz.model
- com.quiz.controllers
- com.quiz.service
- com.quiz.repository
- com.quiz.utils